### PR TITLE
Http support implemented

### PR DESCRIPTION
**Pull request by rockem**
wip

### CODE REVIEW

File: lib/channel.js [hunk 1]
```diff
@@ -1,23 +1,119 @@
...
+        this.callbacks.get(id).call(msg)
       }, { noAck: true })
     } catch (err) {
       throw new ChannelError(err)
     }
   }
+
+  async disconnect() {
+    try {
+      await this.channel.close()
```
Comment (amaziagur) [inline lib/channel.js]: a. i think on the catch you should throw exception or at least log something
b. why you call twice the `await this.connection.close()`?

File: lib/channel.js [hunk 2]
```diff
@@ -1,23 +1,119 @@
...
+
+    }
+  }
+}
+
+const httpInterval = 1000
+
+class HttpTicker {
+  constructor() {
+    this.jobs = {}
```
Comment (amaziagur) [inline lib/channel.js]: i have no idea whats `jobs` is, can you rename it to something meaningful ?

File: lib/channel.js [hunk 3]
```diff
@@ -1,23 +1,119 @@
...
+  }
+
+  updateCallback(id, callback) {
+    this.callbacks[id] = callback
+  }
+
+  async createListener(id, details) {
+    this.jobs[id] = setInterval(async() => {
+      try {
+        const response = await axios.get(details.url)
```
Comment (amaziagur) [inline lib/channel.js]: this can be extracted to a method, to make it more readable

**Review by amaziagur — CHANGES_REQUESTED** [review]

File: lib/channel.js (hunk 1 above)
Comment (rockem) [reply lib/channel.js]: Also you're right right now I'm not catching, we need to add emitters for this kind of thing, so right now I'm not handling it 
logging isn't a good option in js sadly

**Review by amaziagur — APPROVED** [review]

**Pull request merged**